packages/@sanity/default-layout/src/update/PortableTextContent/PortableTextContent.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import {PortableText, PortableTextComponents} from '@portabletext/react'
+import {Box, Card, Code, Heading, Stack, Text} from '@sanity/ui'
+import urlBuilder from '@sanity/image-url'
+import {getProject} from '@sanity/asset-utils'
+import {ImageCard, ListBox, ListBoxOuter, RootBox, SpaceBox} from './PortableTextContent.styles'
+
+interface PortableTextContentProps {
+  value: any
```

Comment:
If we add typings for the block content shape we should make sure this one is also typed correctly